plugins/header_rewrite/conditions.cc

```diff
@@ -1333,4 +1333,77 @@ ConditionSessionTransactCount::append_value(std::string &s, Resources const &res
     TSDebug(PLUGIN_NAME, "Appending SSN-TXN-COUNT %s to evaluation value %.*s", _qualifier.c_str(), length, value);
     s.append(value, length);
   }
-}
\ No newline at end of file
+}
```

Comment:
Is there a missing set of #include's here somewhere ? Don't you need something similar to the include jive from the old tcpinfo plugin?

```
// This is a bit of a hack, to get the more linux specific tcp_info struct ...
#if HAVE_STRUCT_LINUX_TCP_INFO
#include <linux/tcp.h>
#elif HAVE_NETINET_IN_H
#include <netinet/tcp.h>
#endif
```

Of course, if you don't need the linux specific ones, only include the netinet/tcp.h include, but assuming this even compiles, something else is dragging in tcp.h? But depending on that is not good. It seems lulu.h includes netinet/in.h, so maybe add a netinet/tcp.h there too ?